products/sle/main.pm

```diff
@@ -966,7 +966,12 @@ sub load_fips_tests_crypt() {
 sub prepare_target() {
     if (get_var("BOOT_HDD_IMAGE")) {
         if (check_var("BACKEND", "svirt")) {
-            loadtest "installation/bootloader_svirt";
+            if (check_var("ARCH", "s390x")) {
+                loadtest "installation/bootloader_zkvm";
+            }
+            else {
+                loadtest "installation/bootloader_svirt";
+            }
```

Comment:
that's not how it should be, please open a progress issue to improve the two bootloader implementations.

[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).